index.bs

```diff
@@ -622,6 +637,7 @@ be either:
 an {{unrestricted double}} if type is <i>number</i> or <i>date</i>,
 a {{DOMString}} if type is <i>string</i>,
 a list of {{octet}}s if type is <i>binary</i>,
+an integer if the type is <i>bigint</i>,
```

Comment:
This line is a little fudge--the other types have meanings that are more well-defined in terms of Infra or WebIDL, but there isn't a BigInt WebIDL type yet.